### DISCUSSION TITLE
Schedule & Testing Session

### DISCUSSION THREAD

**Discussion by McHouse** [theOehrly/Fast-F1#516] [Q&A] [answered]
preparing the tools for the next preseason tests, I have seen that this:
race_schedule = ff1.get_event_schedule(2023)
includes Pre-Season Testing 2023 in Bahrain
but
race_schedule = ff1.get_event_schedule(2023)
does not include it. Is there another place to look for it?

**Comment by McHouse** [comment]
sorry for the mistake, I correct:

but
race_schedule = ff1.get_event_schedule(2024)
does not include it. Is there another place to look for it?

**Comment by theOehrly** [comment, marked as answer]
I hadn't updated my own schedule backend yet. That one is the only one that supports pre-season testing in advance. Just did that. It should be included now.

Timing and telemetry data will likely be available for the pre-season test (like last year), but that remains to be seen.

Also, the sprint weekend format for this season has changed again. I haven't checked full compatibility with FastF1 yet.

**Comment by McHouse** [comment]
Thank you theOehrly, as always fast and efficient.👍🍻
I read somewhere that they were going to provide the usual LT support.
And when testing the scripts that I have made...⛐ well, you know...😂

Thanks once again 👍😎
